src/bundles/game/types.ts

```diff
@@ -25,13 +25,40 @@ export type GameObject = {
   object: RawGameObject | RawInputObject | RawContainer | undefined;
 };
 
-
 export type GameParams = {
-  scene: Phaser.Scene;
+  scene: Phaser.Scene | undefined;

```

Comment:
Yes, added the undefined so that module functions don't have to be nulled outside of the game room preview. Accessess to the scene are guarded by `mandatory` so will give helpful errors when run outside of the game.